offline/federation_preparation.md

```diff
@@ -0,0 +1,69 @@
+## RabbitMQ
+
+There are two ways to deploy the RabbitMQ cluster:
```

Comment:
```suggestion
There are two methods to deploy the RabbitMQ cluster:
```

One general rule of technical writing is to use one name for one thing. That's usually easier to grasp for readers. So, you should describe here if we're talking about ways or methods.